optimade.rst

```diff
@@ -442,42 +442,62 @@ For example, the following query can be sent to API implementations `exmpl1` and
-In addition to the fields described in the section `property definitions`_, the property definition of a metadata field SHOULD have the key:
+The metadata property is a dictionary of format specified by the field :field:`x-optimade-metadata-definition` in the Property Definition of the field, see `Property Definitions`_.
+Database providers are allowed to define their own metadata properties in :field:`x-optimade-metadata-definition`, but they MUST use the database-specific prefix even for metadata for database-specific fields.
+For example, the metadata property definition of the field :field:`_exmpl_example_field` MUST NOT define a metadata field named, e.g., :field:`accuracy`; the field rather needs to be named, e.g., :field:`_exmpl_accuracy`.
+The reason for this limitation is to avoid name collisions with standard metadata fields defined by the OPTIMADE standard that apply also to database-specific data fields.
 
-- **x-optimade-metadata-for**: This key contains a string with the name of the property for which this metadata property contains the metadata.
-  It MUST be located at the top level of the property definition.
+If an implementation supports the metadata field, it SHOULD return the metadata field whenever the property to which the metadata field belongs is returned.
 
-Example of a returned metadata field:
+Example of a response using the JSON response format with two structure entries that each include a metadata property for the :field:`element_ratios`:
 
-    .. code:: jsonc
+.. code:: jsonc
 
-       {
-         "element_ratios":[0.33336, 0.22229, 0.44425],
-         "element_ratios_meta": {
-           "_exmpl_confidence_interval": [[0.33325,0.33347],[0.22190,0.22268],[0.44392,0.44458]],
-           "_exmpl_confidence_level": 0.95,
+     {
+       "data": [
+         {
+           "type": "structures",
+           "id": "example.db:structs:0001",
+           "attributes": {
+             "element_ratios":[0.33336, 0.22229, 0.44425],
+           },
+           "meta": {
+             "element_ratios": {
+               "_exmpl_confidence_interval": [[0.33325,0.33347],[0.22190,0.22268],[0.44392,0.44458]],
+               "_exmpl_confidence_level": 0.95
+             }
+           }
+         },
+         {
+           "type": "structures",
+           "id": "example.db:structs:1234",
+           "attributes": {
+             "element_ratios":[0.5, 0.5],
+           },
+           "meta": {
+             "element_ratios": {
+               "_exmpl_confidence_interval": [[0.6,0.4],[0.7,0.4]],
+               "_exmpl_confidence_level": 0.90
+             }
+           }
          }
-         //...
-       }
+         // ...
+       ]
+       // ...
+     }
 
-Example of the property definition of a metadata field:
+Example of the corresponding metadata property definition to be placed in the field :field:`x-optimade-metadata-definition` in the property definition of :field:`element_ratios`:
```

Comment:
This sounds a bit strange, as the example property definition for the metadata field below already contains the field `x-optimade-metadata-definition`
```suggestion
Example of the corresponding metadata property definition contained in the field :field:`x-optimade-metadata-definition` which is placed in the property definition of :field:`element_ratios`:
```